### PR TITLE
Theia 1.23, Toolbar, Updater

### LINKED ISSUE #199: [Updater] Error log from updater cannot be opened on Windows due to URI parsing
### Bug Description:
#165 introduced the capability to inspect the update log on error. However, due to parsing the path as an URI, the open log action fails on Windows as there are no opener for the URI that is created from the path. The URI has scheme=c and path=path to log file.

### Steps to Reproduce:

1. Check for updates (for instance in a dev session without a correct update yml present)
2. In the error message, click View Log 
3. Nothing happens, console logs 'No resource handler for...'
4. The URI opened contains a percent-encoded version of the Windows path, i.e C:%5CU

### Additional Information

- Operating System: Windows
- Theia Version: [theia-extensions/theia-blueprint-updater/src/common/updater/theia-updater.ts](https://github.com/eclipse-theia/theia-blueprint/commit/7dc31b0969be5f5b27630022fed88759eb41059d#diff-f47195d665f56013fe6c21a99c06526ad3b5d11382c917d09c9d668c0866c6bfR237)

### Fix
Attached patch ([updater-error-path-as-path.txt](https://github.com/eclipse-theia/theia-blueprint/files/7953834/updater-error-path-as-path.txt)) changes the creation of the URI to parse the path as a path directly, instead of using the URI.parse from vscode-uri. This causes the URI to correctly become a file:/// URI that is openable.

### PR DESCRIPTION
**Pull request by jfaltermeier**
#### What it does
* Updates Theia to 1.23 and upgrades other dependencies
* Enables the new toolbar by default
* fixes an issue with the updater logs (fixes #199)

ksnip_20220225-150609

#### How to test
Build and test blueprint functionality

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Review by vince-fugnitto — COMMENTED** [review]
The changes look good to me, I confirmed that the build, start and packaging work well for me :+1: I added the following [commit](https://github.com/eclipse-theia/theia-blueprint/pull/207/commits/760265705417ea38ae7c5463feb91d1223867000) to fix an issue with the loader.

I have a few comments about the formatting and so on.

File: theia-extensions/theia-blueprint-updater/package.json [hunk 1]
```diff
@@ -46,4 +47,4 @@
     "lint:fix": "eslint --ext js,jsx,ts,tsx src --fix",
     "update:next": "ts-node ../../scripts/update-theia-to-next.ts"
   }
-}
+}
```
[the remark this answers is not in the document]
Comment (jfaltermeier) [reply theia-extensions/theia-blueprint-updater/package.json]: fixed

File: theia-extensions/theia-blueprint-product/package.json [hunk 2]
```diff
@@ -49,4 +49,4 @@
   "peerDependencies": {
     "react": "^16.8.0"
   }
-}
+}
```
[the remark this answers is not in the document]
Comment (jfaltermeier) [reply theia-extensions/theia-blueprint-product/package.json]: fixed

File: theia-extensions/theia-blueprint-updater/src/electron-browser/updater/theia-updater-frontend-contribution.ts [hunk 3]
```diff
@@ -35,6 +35,7 @@ import { ElectronMainMenuFactory } from '@theia/core/lib/electron-browser/menu/e
 import { isOSX } from '@theia/core/lib/common/os';
 import { setInterval, clearInterval } from 'timers';
 import URI from '@theia/core/lib/common/uri';
+import { URI as VSCURI } from 'vscode-uri';
```
[the remark this answers is not in the document]
Comment (jfaltermeier) [reply theia-extensions/theia-blueprint-updater/src/electron-browser/updater/theia-updater-frontend-contribution.ts]: aligned

**Review by vince-fugnitto — APPROVED** [review]
LGTM :+1: Should we squash before merging?

**Comment by jfaltermeier** [comment]
Thanks, I've squashed the commits with the last force-push

**Pull request merged**